bin/wasabi.sh

```diff
@@ -151,13 +152,13 @@ test() {
 resource() {
   for resource in $1; do
     case "${1}" in
-      ui) [ ! -f ./modules/ui/dist/index.html ] && ./bin/build.sh
+      ui) [ ! -f ./modules/ui/dist/index.html ] && build
         ./bin/wasabi.sh status >/dev/null 2>&1 || ./bin/wasabi.sh start
-        open http://$(docker-machine ip wasabi):8080/index.html;;
+        open http://localhost:8080/index.html;;
```

Comment:
Locally the ui is served on :9000 and no where is grunt serve called - I propose changing this line to:
`(cd ./modules/ui && grunt serve)`
it serves the UI and automatically opens the browser at the correct address.